atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/HikariCPConnectionManager.java

```diff
@@ -316,7 +314,11 @@ private State initialState() throws SQLException {
         HikariConfig config = new HikariConfig();
 
         // additional connection properties will go in here
-        Properties props = JdbcConfig.getPropertiesFromDbConfig(dbConfig);
+        Properties props = JdbcConfig.getPropertiesFromDbConfig(
```

Comment:
nit: would like to see this consolidated just like the one extracting the data from DBConfig, could still use the underlying helper and make it private.